### PR TITLE
Table reconfiguration based on demo feedback

### PR DESCRIPTION
**Pull request by mrahanjam**
# Tethering Table reconfiguration

## Description
This PR consolidates the three table components used in tethering into one. Based on demo feedback and requests from product, the tables were converging into one and I asked if I can simply use the same layout across tethering which was okayed. There is less redundancy this way. I also added tooltips to certain columns.

## PR Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [ ] Testing
- [X] General Improvement
- [ ] Cherry Pick

## Links
Jira: [CDAP-18849](https://cdap.atlassian.net/browse/CDAP-18849)

## Screenshots

Screen Shot 2022-02-11 at 3 13 55 PM

### CODE REVIEW

File: app/cdap/components/Administration/TetheringTabContent/shared.styles.tsx [hunk 1]
```diff
@@ -63,8 +63,6 @@ export const GridHeader = styled.div`
 `;
 
 export const GridBody = styled.div`
-  max-height: 300px;
-  overflow: auto;
```
Comment (mrahanjam) [inline app/cdap/components/Administration/TetheringTabContent/shared.styles.tsx:67]: We decided to remove scrolling for now until pagination is done and see if we need scrollable tables.

File: app/cdap/components/Administration/TetheringTabContent/TetheringTable/constants.tsx [hunk 2]
```diff
@@ -57,12 +68,8 @@ export const CONNECTIONS_TABLE_HEADERS = [
     label: T.translate(`${PREFIX}.ColumnHeaders.region`),
   },
   {
-    property: 'omniNamespace',
-    label: T.translate(`${PREFIX}.ColumnHeaders.omniNamespace`),
-  },
-  {
-    property: 'pods',
```
Comment (mrahanjam) [inline app/cdap/components/Administration/TetheringTabContent/TetheringTable/constants.tsx:64]: unused prop/column.

File: app/cdap/components/Administration/TetheringTabContent/TetheringTable/index.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+  GridCell,
+  StyledIcon,
+  LinedSpan,
+} from '../shared.styles';
+import { ICONS, CONNECTIONS_TABLE_HEADERS, PREFIX, DESC_COLUMN_TEMPLATE } from './constants';
+import { IConnection, ITableData } from '../types';
+import { humanReadableDate } from 'services/helpers';
+
+const COLUMN_TEMPLATE = '50px 200px 2fr 1.5fr 1.5fr 200px 1.5fr 120px 120px 225px';
+const REQUEST_DATE_FORMAT = 'MM/DD/YYYY - hh:mm A';
```
Comment (njbriggs) [inline app/cdap/components/Administration/TetheringTabContent/TetheringTable/index.tsx]: Please use `DataFormatter`. I think we should move to a consistent format for all dates/times. Also, formatting in one place will make I18n easier if/when we do that.

Comment (mrahanjam) [reply app/cdap/components/Administration/TetheringTabContent/TetheringTable/index.tsx]: Addressed in [2bcf545f](https://github.com/cdapio/cdap-ui/pull/354/commits/2bcf545f7321480e53d8aea16fa53180404c28d4)

File: app/cdap/components/Administration/TetheringTabContent/TetheringTable/index.tsx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+  StyledIcon,
+  LinedSpan,
+} from '../shared.styles';
+import { ICONS, CONNECTIONS_TABLE_HEADERS, PREFIX, DESC_COLUMN_TEMPLATE } from './constants';
+import { IConnection, ITableData } from '../types';
+import { dateTimeFormat } from 'services/DataFormatter';
+
+const COLUMN_TEMPLATE = '50px 200px 2fr 1.5fr 1.5fr 200px 1.5fr 120px 120px 225px';
+
+const getIconForStatus = (status: string, isForPendingReqs: boolean) => {
```
Comment (seanfeld) [inline app/cdap/components/Administration/TetheringTabContent/TetheringTable/index.tsx:34]: Can we split this file up a bit?

**Review by seanfeld — APPROVED** [review]
just a nit from me

File: app/cdap/components/Administration/TetheringTabContent/TetheringTable/index.tsx (hunk 4 above)
Comment (mrahanjam) [reply app/cdap/components/Administration/TetheringTabContent/TetheringTable/index.tsx:34]: I have a subsequent PR with some changes in this file. I'll try splitting up in that PR if that works for you.

**Pull request merged**